docs/meshstack.metadata-tags.md

```diff
@@ -183,9 +183,7 @@ This is useful if your organization needs additional metadata like e.g. contract
 
 ## Tags on meshUsers
 
-meshUsers are defined globally in meshStack and therefore their tags are identicial within all meshCustomers. As described in [meshPolicies](meshcloud.policies.md#meshpolicies-for-meshUsers/Groups) there are usecases where it makes sense to just apply a set of default tags to all meshUsers. I.e. you want to allow to assign all users to "dev" and "qa" projects, but not to "prod" projects.
-
-<!--snippet:mesh.web.user-->
+meshUsers are defined globally in meshStack and therefore their tags are identical within all meshCustomers. As described in [meshPolicies](meshcloud.policies.md#meshpolicies-for-meshUsers/Groups) there are use cases where it makes sense to just apply a set of default tags to all meshUsers. I.e. you want to allow to assign all users to "dev" and "qa" projects, but not to "prod" projects.
```

Comment:
f: Seems to be an accident. 

(Deleting the `<!--snippet....-->` leads to the snippet not being updated anymore.)

Reply:
indeed. re-addedit.